static.config.js

```diff
@@ -75,6 +75,8 @@ function extractRelevantProjectData (data) {
       followersPrevious,
       dataAgeInDays,
     }
+    if (repo) relevantData.repo = repo
```

Comment:
Here we are allowing the API to give us the web/html URL of the repository instead of having to hardcode it.